content/40.resource-availability.md

```diff
@@ -2,3 +2,4 @@
 
 A manubot instance that hosts version history for this manuscript is available under the Creative Commons Attribution License at <https://github.com/greenelab/iscb-diversity-manuscript>.
 Our analysis of authors and ISCB-associated honorees is available under the Creative Commons Attribution License at <https://github.com/greenelab/iscb-diversity>, with source code also distributed under a BSD 3-Clause License.
+Our Wikipedia name dataset is available under the Creative Commons Attribution License at <https://github.com/greenelab/wiki-nationality-estimate>, with source code also distributed under a BSD 3-Clause License.
```

Comment:
```suggestion
Our Wikipedia name dataset is available under the Creative Commons Attribution License at <https://github.com/greenelab/wiki-nationality-estimate>, with source code to construct the dataset available under a BSD 3-Clause License.
```

Reply:
Note that this is not currently the case. Issue still open at https://github.com/greenelab/wiki-nationality-estimate/issues/3